test/integration/http_subset_lb_integration_test.cc

```diff
@@ -0,0 +1,234 @@
+#include "test/integration/http_integration.h"
+
+#include "absl/strings/str_replace.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace {
+
+const std::string SUBSET_CONFIG = R"EOF(
```

Comment:
I'm going to make fixing this optional since adding an integration test was a bunch of work, but please do prefer using the config helper and associated utilities in future. The more hard coded string config we have in tests, the more we have to update things in 50 places when we change the name of some field, where when we use shared utility programmatic accessors, we only have to update one place. :-)

Reply:
Fair. I think I'd want to add some more helper functions. I'll take that as a for another PR.